src/ophys_etl/decrosstalk/decrosstalk.py

```diff
@@ -306,25 +309,27 @@ def _centered_rolling_mean(data: np.ndarray,
     i0 = window-1
     i1 = window-1+n_t-window
 
-    true_sum = np.zeros(n_t, dtype=float)
-    true_sum[half:n_t-half] = sum_[i0:i1]
-    true_sum[:half] = sum_[i0]
-    true_sum[n_t-half:] = sum_[i1]
-
-    true_sum_sq = np.zeros(n_t, dtype=float)
-    true_sum_sq[half:n_t-half] = sum_sq[i0:i1]
-    true_sum_sq[:half] = sum_sq[i0]
-    true_sum_sq[n_t-half:] = sum_sq[i1]
-
-    true_mask_sum = np.zeros(n_t, dtype=float)
-    true_mask_sum[half:n_t-half] = mask_sum[i0:i1]
-    true_mask_sum[:half] = mask_sum[i0]
-    true_mask_sum[n_t-half:] = mask_sum[i1]
-
-    mean = true_sum/true_mask_sum
-    var = (true_sum_sq/true_mask_sum - mean*mean)
-    var *= (true_mask_sum/(true_mask_sum-1))
-    std = np.sqrt(var)
+    with np.errstate(divide='ignore'):
```

Comment:
similar comment. would be nice for the docstring to mention that `returned arrays can have inf or nan or whatever in the case where xyz happened`

Reply:
Actually, it looks like I'm not suppressing the warning I thought I was. I'll revert this

Reply:
Nevermind. I figured out what I needed. I have added a comment as you requested.